README.md

```diff
@@ -118,6 +119,49 @@ goldrush run reads=long_reads G=3e9
 
 **For more information about the GoldRush algorithm and tips for running GoldRush see our [wiki](https://github.com/bcgsc/goldrush/wiki)** 
 
+# System Requirements
+
+## Hardware Requirements
+
+GoldRush requires a standard computer with enough RAM to support the operations defined by the user. For the assembly of a human genome with a ~60X coverage, GoldRush requires a computer with:

```

Comment:
```
GoldRush requires a standard computer with enough RAM to support the operations defined by the user.
```
Unsure if this is required? I'm not too sure what it meant by it?

Reply:
I guess i meant to say something along the lines of GoldRush requires no special hardware. I'll change this one too